### DISCUSSION TITLE
Rock hunting

### DISCUSSION THREAD

**Discussion by dc1rdb** [Open-Gamma-Project/Mini-SiD#9] [Show and tell]
When looking for radioactive rocks in the wild, I always found it annoying to keep my eyes on the display of whatever gamma detector I was using. Concentrating on small changes in click rate is also less than ideal.
On the other hand, the human brain / ear can easily and subconciously detect very small variations in tone pitch.
So I hooked up the Trigger/TTL output of the Mini SiD to an ESP 8266 programmed as a voltage controlled oscillator:
![circuit](https://github.com/OpenGammaProject/Mini-SiD/assets/9314230/c12baa63-0600-466f-bf91-ce570aeefca9)

The sketch for the ESP 8266 is also very simple, I'm sure there is lots of room for improvement:
```
void setup() {
  pinMode(12, OUTPUT);
}

void loop() {

  int frequencyinput = analogRead(A0);
  frequencyinput = map(frequencyinput, 55, 1023, 100, 5000);
  tone(12, frequencyinput);
  delay(10);
}
```

The output (pin12) of the ESP 8266 is connected to a passive piezo buzzer.

This is the result:
![IMG_20230610_132240](https://github.com/OpenGammaProject/Mini-SiD/assets/9314230/e5ef9912-c9f8-4687-afbd-ae86f4e5900f)

https://github.com/OpenGammaProject/Mini-SiD/assets/9314230/94b5b3d5-aa94-4956-9d66-29ae90044f35

**Comment by NuclearPhoenixx** [comment]
Love the idea and the finished device. How do you get power to the ESP? Are you using batteries or a powerbank?

**Comment by dc1rdb** [comment]
The ESP is a Lolin D1 mini pro with included battery connector and charging circuit. A small Lipo sits between the Mini SiD and the veroboard and the Mini SiD is easily being powered thru the ESP. Battery charging via USB

**Reply by NuclearPhoenixx** [reply]
Oh I see, now that you mentioned it ;)

**Comment by dc1rdb** [comment]
I`ve just finished a unit that incorporates a rotary encoder for threshold adjustment and utilizes a 40 mm dia x 40 mm NaI(Tl) scintillator coupled to a 2x2 SiPM array (MICROFC-60035-SMT-TR). With around four times the scintillator volume compared to my first unit, the sensitivity is significantly improved. 
More details can be found here: https://github.com/dc1rdb/RockFinder
